specification/responses/4xx/get_nhsapp_accounts/429_GetNhsAppAccountTooManyRequests.yaml

```diff
@@ -0,0 +1,23 @@
+description: |+
+  You have made too many requests too quickly, you must send requests at a slower rate.
```

Comment:
I think we should reword this - given the nature of this endpoint the rate limiting happens to the combined total of all requests across all clients so the client that receives the 429 may not have made too many requests. E.g. ClientA sends 500 requests in 60seconds, then ClientB sends 1 request at the end of that 60 seconds window. ClientB would also get a 429.

I'd suggest replacing this with the line from the 429 payload: "This endpoint is currently receiving a high volume of requests and is being rate limited"

Reply:
Updated